tools/tidb-controller.md

```diff
@@ -0,0 +1,110 @@
+---
+title: TiDB Controller User Guide
+category: tools
+---
+
+# TiDB Controller User Guide
+
+TiDB Controller is a command line tool of TiDB, usually used to obtain the status information of TiDB in tuning.
```

Comment:
`tuning` usually means that it could make the database run faster or something. But `tidb-ctl` is a debug tool for most common scenes.